www/js/intro.js

```diff
@@ -102,6 +111,33 @@ angular.module('emission.intro', ['emission.splash.startprefs',
     $state.go('root.main.heatmap');
   };
 
+  $scope.os = function() {
+    if (ionic.Platform.isIOS()) {
+      $scope.OS = 'ios';
+    }
+    else {
+      $scope.OS = 'android';
+    }
+  }
+
+  $scope.fetchItinerumSurvey = function(url) {
+     var data_dict = {
+       'user': {
+         'uuid': $window.device.uuid,
+         'model': $window.device.model,
+         'itinerumVersion': '99c',
+         'os': $scope.OS,
```

Comment:
This should work, but you should test it. I don't remember what the various device settings return, so you should log/debug that and pick and choose what you need.

```
curr_platform: ionic.Platform.platform(),
```

from https://github.com/e-mission/e-mission-phone/blob/master/www/js/splash/storedevicesettings.js#L16
is also an alternative.

Again, **please test**